scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
```

Comment:
Ah, 'kay. I think even Git has a built in flag for detecting things like spaces at the ends of lines and such and can alert you, too... `git diff --check` I need to actually start using this myself, really handy.

Source: https://kparal.wordpress.com/2011/07/04/git-tip-of-the-day-check-for-whitespace-errors-in-diff/